desktop/renderer/remote-component.desktop.js

```diff
@@ -3,22 +3,26 @@ import React, {Component} from 'react'
 import hotPath from '../hot-path'
 import menuHelper from '../app/menu-helper'
 import {injectReactQueryParams} from '../shared/util/dev'
-import {remote, ipcRenderer, screen as electronScreen} from 'electron'
+import electron from 'electron'
 import {resolveRootAsURL} from '../resolve-root'
 import {showDevTools, skipSecondaryDevtools} from '../shared/local-debug.desktop'
 
-const {BrowserWindow} = remote
+const BrowserWindow = electron.BrowserWindow || electron.remote.BrowserWindow
+const ipcRenderer = electron.ipcRenderer
+
 const remoteIdsToComponents = {}
 
 // Remember if we close, it's an error to try to close an already closed window
-ipcRenderer.on('remoteWindowClosed', (event, remoteWindowId) => {
-  if (!remoteIdsToComponents[remoteWindowId]) {
-    return
-  }
+if (ipcRenderer) {
```

Comment:
i believe this is getting pulled into the main thread due to the flow import (hence changing the remote browserwindow stuff also)